src/main/java/uk/gov/digital/ho/hocs/casework/domain/model/Statistic.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.digital.ho.hocs.casework.domain.model;
+
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@NoArgsConstructor
+@AllArgsConstructor
+public class Statistic {
```

Comment:
When I saw the word statistic I got  a bit confused, would DashboardSummary be  a better name?

Reply:
The idea is that this could be used for other purposes, and is a statistic from the database, unsure that `DashboardSummary` would be a great name. In theory it would be good to get this reused for the actual workstacks. Happy to rename, just unsure what a better name would be.

Reply:
Summary?
Aggregate?